### ISSUE TITLE
Cannot singularize "People's"

### ISSUE THREAD

**Issue by weakit** [spencermountain/compromise#965]
`nouns().toSingular()` fails on any text with "People's" in it.

People is correctly singularized to person, but if people's is present, this happens:

```
Uncaught TypeError: Cannot read properties of undefined (reading 'map')
```

Pretty easy to reproduce. Just try:
```js
const doc = nlp("How has chemistry improved people's lives?")
doc.nouns().toSingular()
```

**Comment by spencermountain** [comment]
Thank you. Will look at this today.

**Comment by spencermountain** [comment]
fixed in 14.5.2
Regarding the singular form of that sentence, I don't know what's best. Right now, the handful of Possessives that are technically-plural are ignored in the inflector - `people's right` or `women's bathroom`.  There really are probably 4 or 5, and it's sorta an awkward corner in english - `the indices's point` etc
thanks
